packages/mailer/src/Content.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tempest\Mail;
+
+use Tempest\Mail\Attachment;
+use Tempest\Mail\Exceptions\MailContentWasMissing;
+use Tempest\View\View;
+
+/**
+ * Represents the content of an {@see \Tempest\Mailer\Envelope}.
+ */
+final class Content
+{
+    /**
+     * @param Attachment[] $attachments
+     */
+    public function __construct(
```

Comment:
Still not convinced we need `Content`. I'm ok with Envelope, but I think we need three parameters on the interface:

```php
public Envelope $envelope;
public string|View $content;
public array $attachments;
```

We loose the ability to differentiate between a view path and raw content (what used to be `$html` or `$text`). My proposal is that `$content` always assumes there's a view. Either it's a view object or a path to a view file. 

If we _really_ want to, we could have some kind of `raw` option in the builder which wraps whatever raw content in an instance of `GenericView`. Personally, I don't think we need `raw` atm, and I would just skip it for now, as it can be added later without needing a breaking change